diff -rcNP linux-4.4.3/net/mac80211/mlme.c linux-4.4.3-patched/net/mac80211/mlme.c
*** linux-4.4.3/net/mac80211/mlme.c	2016-02-25 20:01:36.000000000 +0000
--- linux-4.4.3-patched/net/mac80211/mlme.c	2016-03-04 15:02:00.884436169 +0000
***************
*** 410,418 ****
  				      IEEE80211_STA_DISABLE_160MHZ)) ||
  	    !cfg80211_chandef_valid(&chandef)) {
  		sdata_info(sdata,
! 			   "AP %pM changed bandwidth in a way we can't support - disconnect\n",
  			   ifmgd->bssid);
! 		return -EINVAL;
  	}
  
  	switch (chandef.width) {
--- 410,418 ----
  				      IEEE80211_STA_DISABLE_160MHZ)) ||
  	    !cfg80211_chandef_valid(&chandef)) {
  		sdata_info(sdata,
! 			   "AP %pM changed bandwidth in a way we can't support - ignore\n",
  			   ifmgd->bssid);
! 		return 0;
  	}
  
  	switch (chandef.width) {
